tools/cli/commands/project/create.js

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const { exec } = require('node:child_process');
+
+const supportedTemplates = ['webpack', 'vite'];
+
+/**
+ * Validate if the passed template is valid
+ * @param {string} template the name of the template
+ * @returns {boolean}
+ */
+function validateTemplate(template) {
+    const valid = supportedTemplates.includes(template.toLowerCase());
+
+    if (!valid) console.log(chalk.red(`Template ${template} is not a valid option. Please use one of the supported templates - ${supportedTemplates.join(', ')}!`));
+
+    return valid;
+}
+
+/**
+ * Validate if the passed directory exists

```

Comment:
The function setups the directory apart from validating it. Maybe change the description to reflect that.